tests/autobahn/images-for-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.9"
+services:
+  autobahn:
+    build:
+      context: .
+      dockerfile: Dockerfile.autobahn
+      args:
+        USER_ID: ${USER_ID}
+
+  aiohttp:
+    build:
+      context: ../..
+      dockerfile: tests/autobahn/Dockerfile.aiohttp
```

Comment:
Building the base images when running tests is time-consuming. It's best to produce and push them to the GitHub Container Registry. You may draw some inspiration on doing this from https://github.com/ansible/pylibssh/blob/devel/.github/workflows/build-manylinux-container-images.yml.